package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@vscode/jupyter-lsp-middleware",
-    "version": "0.2.29",
+    "version": "0.2.30",
```

Comment:
package-lock.json has the number in it too. You should update that to 30 as well. (I usually do it by hand)

Reply:
okay